### PR TITLE
docs: Update readme for jenny

### PR DESCRIPTION
**Pull request by st-pasha**
# Description

Updating the README, in preparation for the release.

## Checklist

- [x] I have followed the [Contributor Guide] when preparing my PR.
- [-] I have updated/added tests for ALL new/updated/fixed functionality.
- [-] I have updated/added relevant documentation in `docs` and added dartdoc comments with `///`.
- [-] I have updated/added relevant examples in `examples` or `docs`.

## Breaking Change?

- [x] No, this PR is not a breaking change.

[Contributor Guide]: https://github.com/flame-engine/flame/blob/main/CONTRIBUTING.md
[Conventional Commit]: https://conventionalcommits.org
[CHANGELOG]: https://github.com/flame-engine/flame/blob/main/CHANGELOG.md

### CODE REVIEW

File: packages/flame_jenny/jenny/pubspec.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: jenny
-description: Yarn Spinner equivalent for Dart.
-version: 0.0.0
+description: YarnSpinner equivalent for Dart.
+version: 1.0.0
```
Comment (spydon) [inline packages/flame_jenny/jenny/pubspec.yaml:3]: This is done by Melos

File: packages/flame_jenny/jenny/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+## 1.0.0
+
+Initial release; this corresponds to YarnSpinner 2.0 with few extra features.
```
Comment (spydon) [inline packages/flame_jenny/jenny/CHANGELOG.md:4]: This should be done together with the melos reelease, I'll copy this line and add it when I do the release. :)

File: packages/flame_jenny/jenny/README.md [hunk 3]
```diff
@@ -7,5 +7,8 @@ dialogue to users within a game or an app.
 
 The `jenny` library mimics the features and functionality of the [Yarn Spinner] library for Unity.
 
+See Jenny's [official documentation site] for more information.
```
Comment (spydon) [inline packages/flame_jenny/jenny/README.md:10]: Maybe just keep to changes to this file and repurpose the PR

File: packages/flame_jenny/jenny/pubspec.yaml (hunk 1 above)
Comment (st-pasha) [reply packages/flame_jenny/jenny/pubspec.yaml:3]: how would it know that the version should go to 1.0.0 and not to, say, 0.0.1?

Comment (spydon) [reply packages/flame_jenny/jenny/pubspec.yaml:3]: You can choose it for the initial version, or for any version really if you want to break semver which we do constantly. 😅

File: packages/flame_jenny/jenny/README.md (hunk 3 above)
Comment (st-pasha) [reply packages/flame_jenny/jenny/README.md:10]: Ok

**Review by spydon — APPROVED** [review]

**Pull request merged**